api/lttb.md

```diff
@@ -53,6 +58,28 @@ The output for this query:
  2020-04-20 00:00:00+00 | 10.022128489940254
 ```
 
+This example creates a downsampled dataset from multiple series:
+
+```sql
+WITH downsamples AS (
+ SELECT
+  metric_id,
+  toolkit_experimental.lttb("time", value, :datapoint_count) AS samples
```

Comment:
In this code sample, are the values starting with a `:` placeholder parameters? If so, we should change to our normal placeholder parameter format because I found this format very confusing

Reply:
Yeah, I was wondering about that, too.